syntax = "proto3";
package simple-microservice;


// The greeting service definition
service Hello {
    // sends a greeting
    rpc SayHello (SayHelloRequest) returns (SayHelloResponse) {}
} 

message SayHelloRequest {
    string name = 1;
}
message SayHelloResponse {
    string message = 1;
}
